src/store/collections/TransactionsList.java

```diff
@@ -1,5 +1,5 @@
 package store.collections;
 
 public class TransactionsList {
-
-}
+	
+}
```

Comment:
Small thing.....we need a new line at the end of the file past the curly brace for formatting.

Reply:
Oh, thought I fixed that.